### PR TITLE
Fixed the admin adds new case test and added more scenarios

### LINKED ISSUE #958: Fix test admin_adds_new_case_spec.rb
The text is currently not running. Get a test for this functionality running.

### PR DESCRIPTION
**Pull request by erickgurian**
### What github issue is this PR for, if any?
Resolves #958

### What changed, and why?

Fixed the 'admin adds new case' test and added more scenarios

### How will this affect user permissions?
- Volunteer permissions: n/a
- Supervisor permissions: n/a
- Admin permissions: n/a

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
Job done

### CODE REVIEW

File: spec/system/admin_adds_new_case_spec.rb [hunk 1]
```diff
@@ -1,22 +1,66 @@
...
+
+      expect(page.body).to have_content(case_number)
+      expect(page).to have_content('CASA case was successfully created.')
+      expect(page).to have_content('Next Court Date:')
+      expect(page).to have_content('Transition Aged Youth: No')
+    end
+  end
+
+  context 'when the case number field is not filled' do
+    it 'is successful' do
```
Comment (jmkoni) [inline spec/system/admin_adds_new_case_spec.rb:47]: I would change this and L58 to `it 'does not create a new case' do`

**Review by jmkoni — APPROVED** [review]
Approving because my feedback is minor, but, if you have a sec, would love to make the tests more descriptive of what is happening :)

File: spec/system/admin_adds_new_case_spec.rb (hunk 1 above)
Comment (erickgurian) [reply spec/system/admin_adds_new_case_spec.rb]: I thought I had changed those parts. Thanks for the comment, I just corrected it according to your suggestion :)

**Comment by jmkoni** [comment]
@erickgurian rad, i'll merge once the build finishes

**Comment by compwron** [comment]
@erickgurian FYI it looks like your local linter disagrees with the casa project linter about `'` vs `"` no big deal

**Comment by compwron** [comment]
You are invited to the CASA asynchronous retrospective-and-futurespective! Please take a look at https://bit.ly/casaretro (no login required) and add at least one note! Even if it's just "I was here and you are cool" or "omg y'all why do you keep messaging me" we want to hear it. <3
Next week Wednesday 18 November @ 6pm Pacific time we will Zoom about the board and talk about themes and takeaways. (Zoom link in our calendar at http://bit.ly/casacal or slack #casa channel https://rubyforgood.herokuapp.com/ )
Please ESPECIALLY add lots of notes if you won't be able to make it on Wednesday to talk!
* Note: the board will be monitored for spam; message me or any team lead if you see a problem
https://bit.ly/casaretro

**Pull request merged**